orderbook/src/api.rs

```diff
@@ -86,17 +70,41 @@ async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     Ok(err.default_response())
 }
 
-fn wrap_metrics<F>(
-    filter: F,
-    metrics: Arc<Metrics>,
-) -> impl Filter<Extract = (impl Reply,), Error = Rejection> + Clone
-where
-    F: Filter<Extract = (LabelledReply,), Error = Rejection> + Clone + Send + Sync + 'static,
-{
-    warp::any()
-        .and(start_request())
-        .and(filter)
-        .map(move |timer, reply| end_request(metrics.clone(), timer, reply))
+fn handle_metrics(info: warp::log::Info) {
+    let url = info.path();
+
+    let metrics = ApiMetrics::instance();
+    metrics
+        .requests_complete
+        .with_label_values(&[url, info.status().as_str()])
+        .inc();
+    metrics
+        .requests_duration_seconds
+        .with_label_values(&[url])
+        .observe(info.elapsed().as_secs_f64());
```

Comment:
We can't use the path as a log label because some routes have user data in the path. I think this is why we had LabelledRepy wrapper.

Reply:
Is it in the path or in the get parameters?

Reply:
https://protocol-rinkeby.dev.gnosisdev.com/api/ for example `/api/v1/orders/{UID}` is in the path . I would have preferred all parameters in the query but we decided that some routes would have some parameters in the path.